public/docs/_examples/quickstart/ts/index.1.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<!-- #docregion -->
+<html>
+  <head>
+    <title>Hello Angular</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+      body {color:#369;font-family: Arial,Helvetica,sans-serif;}
+    </style>
```

Comment:
In a bare minimal "Hello world" example it seems superfluous to be concerned about style.